Parsers/ASimWebSession/Test/Cisco_Firepower_ASimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,116 @@
+"(2) Info: Missing optional alias [Duration] aliasing non-existent column [NetworkDuration]"
+"(2) Info: Missing optional alias [InnerVlanId] aliasing non-existent column [SrcVlanId]"
+"(2) Info: Missing optional alias [OuterVlanId] aliasing non-existent column [DstVlanId]"
+"(2) Info: Missing optional alias [SessionId] aliasing non-existent column [NetworkSessionId]"
+"(2) Info: Missing optional alias [UserAgent] aliasing non-existent column [HttpUserAgent]"
+"(2) Info: Missing optional field [DstAppId]"
+"(2) Info: Missing optional field [DstAppName]"
+"(2) Info: Missing optional field [DstAppType]"
+"(2) Info: Missing optional field [DstBytes]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DstDeviceType]"
+"(2) Info: Missing optional field [DstDomain]"
+"(2) Info: Missing optional field [DstDvcId]"
+"(2) Info: Missing optional field [DstFQDN]"
+"(2) Info: Missing optional field [DstGeoCity]"
+"(2) Info: Missing optional field [DstGeoCountry]"
+"(2) Info: Missing optional field [DstGeoLatitude]"
+"(2) Info: Missing optional field [DstGeoLongitude]"
+"(2) Info: Missing optional field [DstGeoRegion]"
+"(2) Info: Missing optional field [DstInterfaceGuid]"
+"(2) Info: Missing optional field [DstInterfaceName]"
+"(2) Info: Missing optional field [DstMacAddr]"
+"(2) Info: Missing optional field [DstNatIpAddr]"
+"(2) Info: Missing optional field [DstNatPortNumber]"
+"(2) Info: Missing optional field [DstOriginalUserType]"
+"(2) Info: Missing optional field [DstPackets]"
+"(2) Info: Missing optional field [DstUserId]"
+"(2) Info: Missing optional field [DstVlanId]"
+"(2) Info: Missing optional field [DstZone]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcInboundInterface]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOutboundInterface]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [DvcScope]"
+"(2) Info: Missing optional field [DvcZone]"
+"(2) Info: Missing optional field [EventOriginalResultDetails]"
+"(2) Info: Missing optional field [EventOriginalSubType]"
+"(2) Info: Missing optional field [EventOwner]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [EventSubType]"
+"(2) Info: Missing optional field [FileMD5]"
+"(2) Info: Missing optional field [FileSHA1]"
+"(2) Info: Missing optional field [FileSHA512]"
+"(2) Info: Missing optional field [HttpContentFormat]"
+"(2) Info: Missing optional field [HttpCookie]"
+"(2) Info: Missing optional field [HttpHost]"
+"(2) Info: Missing optional field [HttpIsProxied]"
+"(2) Info: Missing optional field [HttpReferrer]"
+"(2) Info: Missing optional field [HttpRequestBodyBytes]"
+"(2) Info: Missing optional field [HttpRequestCacheControl]"
+"(2) Info: Missing optional field [HttpRequestHeaderCount]"
+"(2) Info: Missing optional field [HttpRequestMethod]"
```

Comment:
We should have this info in Websesion, may be in RequestMethod field in source

Reply:
Not available from source.